styles.css

```diff
@@ -160,3 +161,8 @@ a {
 .calendly[disabled="disabled"] > .logo {
   opacity: 0.5;
 }
+
+/* Overwrites select2 input style */
+.select2-container--default .select2-selection--multiple {
+  border-radius: 5px;
+}
```

Comment:
https://github.com/BobaTalks/bobatalks.github.io/pull/3/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1L83

Might be able to look at/leverage this selector? or at the very least (since css) can we just put this next to the existing select2 styling?